src/relay/transforms/annotate_target.cc

```diff
@@ -52,6 +55,13 @@ class AnnotateTargetWrapper : public ExprMutator {
         return fannotate[op](call->attrs, call->args);
       }
     }
+    if (expr->IsInstance<TupleGetItemNode>()) {
```

Comment:
I have a concern about this. It seems like you have to list all possible nodes here and it's easy to miss something. For now I would suggest adding an exception at least to indicate that we need to add more here.

Reply:
I have a PR ready to add support for TupleNode.

Reply:
I'm not sure yet what we plan on doing with Let nodes (or Ref* nodes). I think just saying they're not supported is sufficient for now though without also throwing an error.